patroni/ctl.py

```diff
@@ -800,3 +805,41 @@ def flush(cluster_name, member_names, config_file, dcs, force, role, target):
                 check_response(r, member.name, 'flush scheduled restart')
             else:
                 click.echo('No scheduled restart for member {0}'.format(member.name))
+
+
+@ctl.command('disable', help='Disable auto failover')
```

Comment:
no, it was not intentional.
We can implement it in the next iteration i.e. pull request